sql-odbc/src/sqlodbc/opensearch_types.c

```diff
@@ -372,7 +372,7 @@ opensearchtype_attr_to_sqldesctype(const ConnectionClass *conn, OID type, int ty
         case SQL_TYPE_DATE:
         case SQL_TYPE_TIME:
         case SQL_TYPE_TIMESTAMP:
-            return SQL_DATETIME;
+            return SQL_TYPE_TIMESTAMP;
```

Comment:
I am not sure this is correct, I believe SQL_DATETIME is the correct return here?

Reply:
This has be reverted in [1817773](https://github.com/Bit-Quill/opensearch-project-sql/pull/15/commits/1817773228666e95818487f06aecdaba7caab8eb)